clients/fossid-webapp/src/main/kotlin/model/report/Model.kt

```diff
@@ -21,11 +21,17 @@ package org.ossreviewtoolkit.clients.fossid.model.report
 
 /**
  * Available FossID report types.
+ * FossID accepts the following report types:
```

Comment:
I appreciate the improved docs, but IMO they should be split out to a separate commit, and then really only the default format should be changed in this commit.